lib/Auth/Process/COmanageDbClient.php

```diff
@@ -606,12 +622,28 @@ static function ($verified, $mail) {
                 }
             }
             if (!empty($attributes['edupersonscopedaffiliation'])) {
-                $state['Attributes']['eduPersonScopedAffiliation'] = explode(
+                $ePSA = explode(
                     ',',
                     $attributes['edupersonscopedaffiliation']
                 );
                 // XXX Remove any duplicate edupersonscopedaffiliation
-                $state['Attributes']['eduPersonScopedAffiliation'] = array_filter(array_unique($state['Attributes']['eduPersonScopedAffiliation']));
+                $ePSA = array_filter(array_unique($ePSA));
+                // XXX Remove all non schac containing entries
+                $ePSA = array_filter(
+                    $ePSA,
+                    static function ($value) {
+                        return (strpos($value, "schac:") !== false);
+                    }
+                );
+                // XXX Remove schac: prefix
+                $ePSA = array_map(
+                    static function ($value) {
+                        return str_replace('schac:', '', $value);
+                    }, $ePSA
+                );
+                if(!empty($ePSA)) {
+                  $state['Attributes']['eduPersonScopedAffiliation'] = $ePSA;
+                }
```

Comment:
Remove ePSA and add TODO Aggregate all scoped affiliation from linked org identities